likes.rb

```diff
@@ -0,0 +1,18 @@
+def likes(names)
+
+  if names.empty?
+    puts "No one likes this."
+  end
+
+  if names.count == 1
+    puts "#{names.join(' ')} likes this."
+  end
+
+  if names.count == 2
+    puts "#{names.join(' and ')} like this."
+  end
+
+  if names.count == 3
+    puts "#{names.join(', ')} like this."
```

Comment:
To get this output `Jem, Lucas, and Mickey like this."` you can take this approach.

Join everything in the array _except_ the last element. For example. `names[0...-1].join(", ")`

And then, just concatenate the last element of the array. ` ... + "and names[-1] like this."`

Reply:
Remember that the `...` range syntax does _not_ include the second index, unlike `..` which includes it.